resources/views/print/print.blade.php

```diff
@@ -1,26 +1,24 @@
 <div class="card">
-    <div class="card-header">@lang('print.printing_account')</div>
+    <div class="card-header">@lang('print.print_document')</div>
     <div class="card-body">
-        <form class="form-horizontal" role="form" method="POST" action="{{ route('print') }}" enctype="multipart/form-data">
+        <form class="form-horizontal" role="form" method="POST" action="{{ route('print.print') }}" enctype="multipart/form-data">
             @csrf
-            <input type="hidden" name="_method" value="PUT">
+            @method('PUT')
             <div class="row">
-                <div class="col-md-6">
-                    <label  class="control-label" for="file_name">@lang('print.document')</label>
-                </div>
+                <div class="col-md-6"> @lang('print.select_document')</div>
+                <div class="col-md-3 text-right"> @lang('print.twosided')</div>
+                <div class="col-md-3"> <input type="checkbox"  name="two_sided" id="two_sided"  checked/> </div>
             </div>
-            <div class="row" style="margin-top:10px;">
-                <div class="col-md-6">
-                    <input type="file"  name="file_to_upload" id="file_to_upload">
-                </div>
-                <div class="col-md-3">
-                    <input type="checkbox"  name="two_sided" id="two_sided"  checked/> @lang('print.twosided')
-                </div>
+            <div class="row">
+                <div class="col-md-6"><input type="file"  name="file_to_upload" id="file_to_upload" required></div>
+                <div class="col-md-3 text-right">  @lang('print.number_of_copies') </div>
```

Comment:
Definition for translations.

Reply:
Please grep for all `@lang(...)` instances, and check if each of them has English and Hungarian translations in the same order. I won't issue any more comments like this.

Reply:
Yeah, I forgot the translations, will add them.